src/components/PortfolioProject/PortfolioProjectMobile.tsx

```diff
@@ -0,0 +1,122 @@
+import React, { FC } from 'react';
+
+import { Tag } from '../Tag/Tag';
+import { PortfolioProjectDialogProps } from './PortfolioProject';
+import { IconButton } from '../IconButton/IconButton';
+import { Button } from '../Button/Button';
+
+import { X, ChevronLeft, ChevronRight, Code, Image, ExternalLink } from 'react-feather';
+import { Box, Dialog, makeStyles, Typography } from '@material-ui/core/';
+
+const useStyles = makeStyles((theme) => ({
+  iconButtons: {
+    backgroundColor: theme.palette.background.paper,
+    padding: theme.spacing(1, 2),
+    display: 'flex',
+    flexDirection: 'row',
+    justifyContent: 'space-between',
+    opacity: 0.4,
+  },
+  img: {
+    marginTop: theme.spacing(1, 2),
+    width: '100%',
+    height: 'auto',
+  },
+  header: {
```

Comment:
Situation like in `src/components/PortfolioProject/PortfolioProjectDesktop.tsx`.If you have less than four styles you can always pass them directly into the Box component as props